NTupleAnalyzer_nanoAOD.py

```diff
@@ -2312,14 +2321,15 @@ def GetLLJJMasses3Jets(l1,l2,j1,j2,j3):
 	
 	if (Branches['Pt_muon1'][0] < 45): continue
 	if (Branches['Pt_muon2'][0] < 45): continue #this should be removed if TTBarDataDriven will be used
-	if nonisoswitch != True:
-			if (Branches['Pt_muon2'][0] < 45) and (Branches['Pt_miss'][0] < 45): continue
+	#if nonisoswitch != True:
+	#		if (Branches['Pt_muon2'][0] < 45) and (Branches['Pt_miss'][0] < 45): continue
 	if (Branches['Pt_jet1'][0] <  45): continue
-	#if (Branches['Pt_jet2'][0] <  45): continue #fixme turned off for qcd check.....turn back on!
-	if (Branches['St_uujj'][0] < 275) and (Branches['St_uuj'][0] < 225): continue
+	if (Branches['Pt_jet2'][0] <  45): continue #fixme turned off for qcd check.....turn back on!
+	if (Branches['St_uujj'][0] < 275): continue
+	#if (Branches['St_uujj'][0] < 275) and (Branches['St_uuj'][0] < 225): continue
```

Comment:
Turn this back on, and comment out line 2328. This is to include the single LQ channel

Reply:
Right. I think that variable hadn't been added at some point, so I started commenting it out. I see it's there now, though.